src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerGovernmentOrPublicAuthorityValidator.java

```diff
@@ -112,11 +112,9 @@ private boolean validateStartDate(LocalDate startDate, Errors errors, String log
                 && DateValidators.isDateInPast(startDate, qualifiedFieldName, errors, loggingContext);
     }
 
-
-
     private boolean validateNatureOfControl(List<NatureOfControlType> fields, Errors errors, String loggingContext) {
         String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.BENEFICIAL_OWNERS_GOVERNMENT_OR_PUBLIC_AUTHORITY_FIELD, NATURE_OF_CONTROL_FIELDS);
-        return NatureOfControlValidators.checkAtLeastOneFieldHasValue(fields, qualifiedFieldName, errors, loggingContext);
+        return NatureOfControlValidators.checkAtLeastOneFieldHasValue(fields, null, qualifiedFieldName, false, errors, loggingContext);
```

Comment:
This is temporary as this will change when we get to work on these bo types. Can use a wrapper if preferred.

Reply:
We could have an overloaded method in the `NatureOfControlValidators` class - one that takes two lists and another that takes just the one. I'm not a big fan of passing `nulls` in to methods (unless method is private).